lib/bowling.rb

```diff
@@ -0,0 +1,39 @@
+class Bowling
+  attr_reader :total_score, :frame
+
+  def initialize
+    @total_score = 0
+  end
+
+  def total_frames(frame_scores)
+    raise 'Minimum of 10 frames must be entered' if frame_scores.count < 10
+    # p frame_scores
+    frame_scores.map.with_index do |frame, index|
+      unless [9, 10, 11].include?(index)
```

Comment:
Magic Numbers... consider using constants when needing to refer to numbers. This gives the numbers meaning and helps the code read better